MatrixKit/Controllers/MXKCallViewController.m

```diff
@@ -533,9 +547,16 @@ - (void)call:(MXCall *)call stateDidChange:(MXCallState)state reason:(MXEvent *)
             [localPreviewActivityView startAnimating];
             break;
         case MXCallStateCreateOffer:
-            self.isRinging = YES;
+        {
+            BOOL isCallKitAvailable = [MXCallKitAdapter callKitAvailable] && [MXKAppSettings standardAppSettings].isCallKitEnabled;
```

Comment:
Please add a comment explaining why, in case of CallKit, we delay the ringback to the call of handleAudioSessionActivationNotification.
The explanation in the PR comment is pretty clear and can be c+p here.